app/code/community/Oyst/OneClick/Block/OneClick.php

```diff
@@ -100,21 +100,24 @@ public function oneClickButtonPickToFirstAddToCartButtons()
     }
 
     /**
-     * Change
+     * Add custom styles to button
      *
-     * @return stringw
+     * @return string
      */
-    public function getButtonWrapperPosition()
+    public function addButtonWrapperStyles()
```

Comment:
This function goal is to create a block:
```
<style>
    [content of the backend textarea]
</style>
```
and be called in `oneclick.phtml` before `<div id="oyst-1click-button-wrapper" ... >`